whois-commons/src/main/resources/internals_schema.sql

```diff
@@ -71,6 +71,28 @@ CREATE TABLE `default_maintainer_sync_history` (
     `is_synchronised` tinyint(1) DEFAULT 0,PRIMARY KEY (`id`)
 ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
 
+
+DROP TABLE IF EXISTS `default_maintainer`;
+CREATE TABLE `default_maintainer` (
+    `id` int(10) UNSIGNED NOT NULL AUTO_INCREMENT,
+    `org` varchar(256) NOT NULL,
+    `mntner` varchar(256) NOT NULL,
+    `timestamp` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+    `uuid` varchar(256) NOT NULL,
+    `email` varchar(256),
+    PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+
+DROP TABLE IF EXISTS `default_maintainer_sync`;
```

Comment:
We need to script the pre-population of the sync table from the sync_history table

Reply:
Yes there are insert statements in a patch file with this PR

Reply:
Thanks